src/utils/binder.js

```diff
@@ -0,0 +1,21 @@
+import { bindKey, rest } from 'lodash';
+
+export default class Binder {
+  constructor() {
+    this.disposal = [];
+  }
+
+  on(emitter, event, handler) {
```

Comment:
I suppose that some event emitters support a thirst `capturing` argument, I found the spread approach a lot less clear to read, but you're right that it's worked fine in it's current incarnation. Might as well keep it.